app/assets/javascripts/tooltip.js

```diff
@@ -0,0 +1,19 @@
+var showTooltips = function(options) {
```

Comment:
@floriank `showTooltips` is used [here](https://github.com/opf/openproject/pull/3238/files#diff-b970740d79353d77d89b9c8f6fbdaa90R38)